FirebaseAuth/Sources/Public/FirebaseAuth/FirebaseAuth.h

```diff
@@ -51,3 +51,4 @@
 #import "FIRPhoneMultiFactorInfo.h"
 #import "FIRTOTPMultiFactorAssertion.h"
 #import "FIRTOTPMultiFactorGenerator.h"
+#import "FIRTOTPSecret.h"
```

Comment:
It looks likes there are two additional new public headers in https://github.com/firebase/firebase-ios-sdk/tree/totp-release-0413/FirebaseAuth/Sources/Public/FirebaseAuth.  Were those two headers' APIs approved as well? If so, they should also be here.

Reply:
Yes

```
#import "FIRTOTPMultiFactorAssertion.h"
#import "FIRTOTPMultiFactorGenerator.h"
#import "FIRTOTPSecret.h"
```

are the three headers present in this file. I believe, `FIRTOTPSecret` was the only one added in this PR, the other two were already existing.

Reply:
I see. Thanks.